.travis.yml

```diff
@@ -1,15 +0,0 @@
-language: node_js # don't install any environment
-
-node_js:
-- "lts/*"
-
-script: npm run coverage
-
-after_script:
-- if [[ "$TRAVIS_OS_NAME" == "linux" &&
-    (( -n "$TRAVIS_PULL_REQUEST" && "$TRAVIS_PULL_REQUEST" != "false" ) || "$TRAVIS_BRANCH" == "master" ) ]]; then
-    npm run report-coveralls;
```

Comment:
We can now remove the `report-coveralls` npm script, right?

Reply:
Done